bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1102,7 +1102,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
-                image: quay.io/argoprojlabs/argocd-operator:v0.6.0
+                image: quay.io/argoprojlabs/argocd-operator@sha256:99aeec24cc406d06d18822347d9ac3ed053a702d8419191e4e681075fed7b9bb
```

Comment:
@reginapizza Is there a reason we want to move away from the v0.6.0 floating tag here?

Reply:
I wasn't entirely sure that was right... that was an automatic change that was made when following the steps from the [release process](https://argocd-operator.readthedocs.io/en/latest/release-process/) :/